bin/GCStress/GCStress.vcxproj

```diff
@@ -31,8 +31,10 @@
         $(ChakraCommonMemoryLib);
         $(ChakraRuntimePlatformAgnostic);
         $(ChakraCommonLinkDependencies);
+        Ole32.lib;
```

Comment:
Why do we need this dependency?

Reply:
IIRC, it was because we introduced use of `CoCreateGuid`

Reply:
Can't we just use the activityId? 

---
In reply to: [187488631](https://github.com/Microsoft/ChakraCore/pull/5119#discussion_r187488631) [](ancestors = 187488631)

Reply:
Nevermind. I guess we don't always have an activityId.  

---
In reply to: [187649237](https://github.com/Microsoft/ChakraCore/pull/5119#discussion_r187649237) [](ancestors = 187649237,187488631)

Reply:
yes, Recycler instances span activity IDs.  ScriptContext telemetry data will include the `RecyclerID` guid for correlation with an `activityID`